Getting this error: Module not found: Error: Can't resolve 'core-js-pure/features/global-this'

When trying to run dev server getting this error
ERROR in ./packages/lib-build-tools/node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js 2:17-61
Module not found: Error: Can't resolve 'core-js-pure/features/global-this' in '/Users/PATH_TO_CODEBASE/frontend/packages/lib-build-tools/node_modules/@pmmmwh/react-refresh-webpack-plugin/client'

--- a/docs/TROUBLESHOOTING.md
+++ b/docs/TROUBLESHOOTING.md
@@ -53,6 +53,14 @@ you can fallback to use the legacy dependency resolution algorithm and it should
 npm install --legacy-peer-deps
 ```
 
+## Usage with CSS Files/Imports
+
+This plugin does not provide HMR for CSS. To achieve that,
+you should be using [`style-loader`](https://github.com/webpack-contrib/style-loader) or [`mini-css-extract-plugin`](https://github.com/webpack-contrib/mini-css-extract-plugin).
+
+Both provides HMR capabilities out of the box for Webpack 5 -
+if are still on Webpack 4 and uses `mini-css-extract-plugin`, you might have to [do some setup](https://github.com/webpack-contrib/mini-css-extract-plugin/#hot-module-reloading-hmr).
+
 ## Usage with Indirection (like Workers and JS Templates)
 
 If you share the Babel config for files in an indirect code path (e.g. Web Workers, JS Templates with partial pre-render) and all your other source files,

--- a/docs/TROUBLESHOOTING.md
+++ b/docs/TROUBLESHOOTING.md
@@ -55,7 +55,8 @@ npm install --legacy-peer-deps
 
 ## Usage with CSS Files/Imports
 
-This plugin does not provide HMR for CSS. To achieve that,
+This plugin does not provide HMR for CSS.
+To achieve that,
 you should be using [`style-loader`](https://github.com/webpack-contrib/style-loader) or [`mini-css-extract-plugin`](https://github.com/webpack-contrib/mini-css-extract-plugin).
 
 Both provides HMR capabilities out of the box for Webpack 5 -

--- a/package.json
+++ b/package.json
@@ -58,7 +58,7 @@
   "dependencies": {
     "ansi-html-community": "^0.0.8",
     "common-path-prefix": "^3.0.0",
-    "core-js-pure": "^3.8.1",
+    "core-js-pure": "^3.23.3",
     "error-stack-parser": "^2.0.6",
     "find-up": "^5.0.0",
     "html-entities": "^2.1.0",

